docs/man/git-lfs-config.5.ronn

```diff
@@ -56,6 +56,10 @@ be scoped inside the configuration for a remote.
   Enables in-memory SSH and Git Credential caching for a single 'git lfs'
   command. Default: false. This will default to true in v2.1.0.
 
+* `lfs.storage`
+
+  Allow override LFS storage directory. Default: lfs in Git repository directory.
```

Comment:
It may also be worth mentioning that you should not run `git lfs prune` if you have different repositories sharing the same directory.